aspnet/fundamentals/static-files.rst

```diff
@@ -117,10 +117,10 @@ In order for the user to browse the ``MyStaticFiles`` directory, you can configu
 
 At this point, if the user enters an address of ``http://<yourApp>/StaticFiles``, the browser will display the files in the ``MyStaticFiles`` directory.
 
-Serving default files
+Serving a default document
 ---------------------
 
-In order for your Web app to serve a default page without the user having to fully qualify the URI, call the ``UseDefaultFiles`` extension method from ``Startup.Configure`` as follows. Note that you must still call ``UseStaticFiles`` as well. This is because ``UseDefaultFiles`` is a `URL re-writer` that doesn't actually serve the file. You must still specify middleware (``UseStaticFiles``, in this case) to serve the file.
+Setting a default home page gives site visitors a place to start when visiting your site. Without a default site users will see a blank page unless they enter a fully qualified url to a document.  In order for your Web app to serve a default page without the user having to fully qualify the URI, call the ``UseDefaultFiles`` extension method from ``Startup.Configure`` as follows. Note that you must still call ``UseStaticFiles`` as well. This is because ``UseDefaultFiles`` is a `URL re-writer` that doesn't actually serve the file. You must still specify middleware (``UseStaticFiles``, in this case) to serve the file.
```

Comment:
> Note that you must still call ``UseStaticFiles`` as well. This is because ``UseDefaultFiles`` is a `URL re-writer` that doesn't actually serve the file. You must still specify middleware (``UseStaticFiles``, in this case) to serve the file. 

You have this same note below the following code snippet, which feels repetitive. Let's move this text into the note below and remove this sentence here.